src/openfl/display/DisplayObject.hx

```diff
@@ -428,7 +433,7 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 			
 		}
 		
-		if (event.bubbles && parent != null && parent != this) {
+		if (tmpParent != null && tmpParent != this) {
```

Comment:
Yes, the only difference is that we temporary store the parent. 
Due to "super.__dispatchEvent (event);" and whatever happens there, it might cause the parent to be removed, thus the following condition / code is not executed anymore. When storing it temporary, we can still access it and execute the condition as desired (continue bubbling).